src/tls.c

```diff
@@ -878,7 +883,7 @@ static int Hmac_HashFinalRaw(Hmac* hmac, unsigned char* hash)
  */
 static int Hmac_OuterHash(Hmac* hmac, unsigned char* mac)
 {
-    int ret = BAD_FUNC_ARG;
+    int ret;
```

Comment:
Same here

Reply:
it's fine not to initialize that one but also harmless to initialize it (to ` = WC_NO_ERR_TRACE(BAD_FUNC_ARG)`).  going with the latter, at your suggestion.  that's already what I went with in nearly every other case like this.